### PR TITLE
Issue credential requests

### PR DESCRIPTION
**Pull request by ilanolkies**
- [x] Issued credential requests reducer - cherry picked from #47 
- [x] Credential requests entity
- [x] Operations
  - [x] Init issued credential requests
  - [x] Issue credential request
  - [x] Set issued credential request status
  - [x] Delete issued credential request

### CODE REVIEW

**Review by jessgusclark — APPROVED** [review]

**Comment by ilanolkies** [comment]
- The reducer file has name `issuedCredentialReuqests` because future work should add `receivedCredentialRequests`
- The operation file has name `credentailRequests` to maintain the request-receive protocol in a single file
- The entity is `CredentialRequest` but may change when receiving feature is added.

File: packages/rif-id-core/test/util.ts [hunk 1]
```diff
@@ -137,3 +153,17 @@ export const issueTestCredential = async (subject: string): Promise<Daf.Credenti
...
+  const app = express()
+  app.use(bodyParser.text())
+
+  app.post('/request_credential', function(req, res) {
+    const jwt = req.body
+    res.status(200).send('')
+  })
+
+  return app.listen(handle)
+}
```
Comment (javiesses) [inline packages/rif-id-core/test/util.ts:169]: Maybe in the future we can mock `fetch` package instead of creating a new server

**Review by javiesses — COMMENTED** [review]
LRGTM

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "@types/jest": "^26.0.0",
     "@typescript-eslint/eslint-plugin": "^3.3.0",
     "@typescript-eslint/parser": "^3.3.0",
+    "debug": "^4.1.1",
```
[the remark this answers is not in the document]
Comment (ilanolkies) [reply package.json:29]: It is handy for

```
import Debug from 'debug'
Debug.enable('*')
```

to see DAF logs

File: packages/rif-id-core/test/util.ts (hunk 1 above)
Comment (ilanolkies) [reply packages/rif-id-core/test/util.ts:169]: Opened #51

**Review by javiesses — APPROVED** [review]

**Pull request merged**